app/controllers/api/v1/accounts/macros_controller.rb

```diff
@@ -34,6 +34,12 @@ def update
     end
   end
 
+  def execute
+    ::MacrosExecutionJob.perform_later(@macro, conversation_ids: params[:conversation_ids], user: current_user)
```

Comment:
let's use `Current.user` instead. We are trying to remove the usage `current_user` in favor of this.